### ISSUE TITLE
Drupal 10 is broken; Drush 12 does not work with Drush Launcher

### ISSUE THREAD

**Issue by rosiel** [Islandora-Devops/isle-buildkit#283]
Drush Launcher is a tool that adds the path of `drush` to your path so commands can be written as `drush` (rather than `vendor/bin/drush`). It is known to be [incompatible](https://github.com/drush-ops/drush-launcher/issues/105) with Drush 12 (required by Drupal 10). 

The Site Template invokes `drush` so it is failing.

Suggestion: Add /var/www/drupal/vendor/bin to $PATH. I'm not sure how to do this, I have tried both in the Dockerfile and in install.sh and neither of them work. The drupal logs report: 

```
d10-rosie-drupal-dev-1  | [03-Aug-2023 14:57:26 UTC] PHP Fatal error:  Uncaught RuntimeException: Drush Launcher is not compatible with Drush 12+. See https://github.com/drush-ops/drush-launcher/issues/105. in /var/www/drupal/vendor/drush/drush/includes/preflight.inc:23
d10-rosie-drupal-dev-1  | Stack trace:
d10-rosie-drupal-dev-1  | #0 phar:///usr/bin/drush/bin/drush.php(141): drush_main()
d10-rosie-drupal-dev-1  | Islandora-Devops/isle-site-template#1 /usr/bin/drush(10): require('phar:///usr/bin...')
d10-rosie-drupal-dev-1  | Islandora-Devops/isle-site-template#2 {main}
d10-rosie-drupal-dev-1  |   thrown in /var/www/drupal/vendor/drush/drush/includes/preflight.inc on line 23
d10-rosie-drupal-dev-1  | s6-rc: warning: unable to start service install: command exited 128
d10-rosie-drupal-dev-1  | /run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
d10-rosie-drupal-dev-1  | prog: fatal: stopping the container.
```
and the container has stopped itself, so I can't get in to fix it.

**Comment by rosiel** [comment]
Drush is installed in Buildkit and /usr/bin/drush is assumed here:
* https://github.com/Islandora-Devops/isle-buildkit/blob/main/drupal/rootfs/usr/local/bin/drush

Drush is called without a path here:
* https://github.com/Islandora-Devops/isle-buildkit/blob/main/drupal/rootfs/usr/local/bin/install-drupal-site.sh#L173
* and here: https://github.com/Islandora-Devops/isle-buildkit/blob/main/drupal/rootfs/etc/islandora/utilities.sh
* and here: test/rootfs/etc/s6-overlay/scripts/install.sh
* and a few times in the code-server.

also /usr/share/drush is listed here:
* https://github.com/Islandora-Devops/isle-buildkit/blob/main/drupal/rootfs/etc/drush/drush.yml

**Comment by rosiel** [comment]
Solved with #286
